eng/common/scripts/Update-DocsMsMetadata.ps1

```diff
@@ -129,13 +129,12 @@ author: $author
 ms.author: $msauthor
 ms.date: $date
 ms.topic: reference
-ms.prod: azure
-ms.technology: azure
 ms.devlang: $Language
 ms.service: $service
 ---
 "@
 
+  $ReadmeContent = $ReadmeContent -replace "https://docs.microsoft.com(/en-us)?", ""
```

Comment:
This would erase links whose value is `https://docs.microsoft.com` or  `https://docs.microsoft.com/en-us` ... 

Running this in the console gives: 

```pwsh
 @"
https://docs.microsoft.com/
https://docs.microsoft.com

https://docs.microsoft.com/en-us/
https://docs.microsoft.com/en-us

https://docs.microsoft.com/test/
https://docs.microsoft.com/test

https://docs.microsoft.com/en-us/test/
https://docs.microsoft.com/en-us/test
"@ -replace "https://docs.microsoft.com(/en-us)?", ""
/
    # <-- there should be a / here

/
    # <-- there should be a / here

/test/
/test

/test/
/test
```

Another approach might be to include the trailing forward slash: 

```suggestion 
  $ReadmeContent = $ReadmeContent -replace "https://docs.microsoft.com(/en-us)?/?", "/"
```

Running in the console gives: 

```pwsh
@"
https://docs.microsoft.com/
https://docs.microsoft.com

https://docs.microsoft.com/en-us/
https://docs.microsoft.com/en-us

https://docs.microsoft.com/test/
https://docs.microsoft.com/test

https://docs.microsoft.com/en-us/test/
https://docs.microsoft.com/en-us/test
"@ -replace "https://docs.microsoft.com(/en-us)?/?", "/"
/
/

/
/

/test/
/test

/test/
/test
```